store/postgres/src/deployment_store.rs

```diff
@@ -38,6 +38,8 @@ use crate::relational_queries::FromEntityData;
 use crate::{connection_pool::ConnectionPool, detail};
 use crate::{dynds, primary::Site};
 
+const POSTGRES_MAX_BINDINGS_PER_STATEMENT: usize = u16::MAX as usize; // 65535
```

Comment:
It might be better to call this `POSTGRES_MAX_PARAMETERS` since that's what the Postgres error message complains about.